filogic: set R4-Pro 8X combo copper link and activity LEDs

Use netdev modes: green for link and yellow for RX/TX activity.
The companion software-activity patch sets the yellow LED interval.
The SFP overlays disable the corresponding
PHY LED nodes, so these defaults cannot light an unused copper jack.

--- a/target/linux/mediatek/filogic/base-files/etc/board.d/01_leds
+++ b/target/linux/mediatek/filogic/base-files/etc/board.d/01_leds
@@ -62,6 +62,12 @@
 bananapi,bpi-r4-lite)
 	ucidef_set_led_netdev "sfp0" "sfp0" "green:sfp" "sfp0" "link tx rx"
 	;;
+bananapi,bpi-r4-pro-8x)
+	ucidef_set_led_netdev "eth6_link" "ETH6 Link" "mdio-bus:18:green:lan" "mxl_lan4" "link"
+	ucidef_set_led_netdev "eth6_activity" "ETH6 Activity" "mdio-bus:18:yellow:lan" "mxl_lan4" "tx rx"
+	ucidef_set_led_netdev "eth7_link" "ETH7 Link" "mdio-bus:1c:green:wan" "eth1" "link"
+	ucidef_set_led_netdev "eth7_activity" "ETH7 Activity" "mdio-bus:1c:yellow:wan" "eth1" "tx rx"
+	;;
 bazis,ax3000wm)
 	ucidef_set_led_netdev "wanact" "WANACT" "mdio-bus:05:amber:wan" "eth0" "rx tx"
 	ucidef_set_led_netdev "wanlink" "WANLINK" "mdio-bus:05:green:wan" "eth0" "link"
